marketplace-and-integrations/integrations/categories.md

```diff
@@ -22,7 +22,7 @@ Umbraco HQ has developed and made available integrations with different SaaS (So
+<table data-card-size="large" data-view="cards"><thead><tr><th></th><th></th><th data-hidden data-card-target data-type="content-ref"></th><th data-hidden data-card-cover data-type="files"></th></tr></thead><tbody><tr><td><strong>ActiveCampaign</strong></td><td>A form picker and rendering component for ActiveCampaign forms.</td><td><a href="activecampaign.md">activecampaign.md</a></td><td><a href="../.gitbook/assets/activecampaign (1).png">activecampaign (1).png</a></td></tr><tr><td><strong>ActiveCampaign with Umbraco Forms</strong></td><td>A custom workflow that allows form entries to be mapped to an ActiveCampaign contact and is stored within the Customer Relationship Management (CRM) platform.</td><td><a href="activecampaign.md">activecampaign.md</a></td><td><a href="../.gitbook/assets/activecampaign (1).png">activecampaign (1).png</a></td></tr><tr><td><strong>Dynamics</strong></td><td>A form picker and rendering component for Dynamics 365 forms.</td><td><a href="dynamics.md">dynamics.md</a></td><td><a href="../.gitbook/assets/dynamics (1).png">dynamics (1).png</a></td></tr><tr><td><strong>Hubspot</strong></td><td>A form picker and rendering component for Hubspot forms.</td><td><a href="hubspot.md">hubspot.md</a></td><td><a href="../.gitbook/assets/hubspot.png">hubspot.png</a></td></tr><tr><td><strong>Hubspot with Umbraco Forms</strong></td><td>A custom workflow that allows form entries to be mapped to a HubSpot contact record, and then stored within the CRM platform.</td><td><a href="hubspot-with-umbraco-forms.md">hubspot-with-umbraco-forms.md</a></td><td><a href="../.gitbook/assets/hubspot.png">hubspot.png</a></td></tr></tbody></table>
```

Comment:
warning shows that it needs to be defined on last sentence as well 💡 
```suggestion
<table data-card-size="large" data-view="cards"><thead><tr><th></th><th></th><th data-hidden data-card-target data-type="content-ref"></th><th data-hidden data-card-cover data-type="files"></th></tr></thead><tbody><tr><td><strong>ActiveCampaign</strong></td><td>A form picker and rendering component for ActiveCampaign forms.</td><td><a href="activecampaign.md">activecampaign.md</a></td><td><a href="../.gitbook/assets/activecampaign (1).png">activecampaign (1).png</a></td></tr><tr><td><strong>ActiveCampaign with Umbraco Forms</strong></td><td>A custom workflow that allows form entries to be mapped to an ActiveCampaign contact and is stored within the Customer Relationship Management (CRM) platform.</td><td><a href="activecampaign.md">activecampaign.md</a></td><td><a href="../.gitbook/assets/activecampaign (1).png">activecampaign (1).png</a></td></tr><tr><td><strong>Dynamics</strong></td><td>A form picker and rendering component for Dynamics 365 forms.</td><td><a href="dynamics.md">dynamics.md</a></td><td><a href="../.gitbook/assets/dynamics (1).png">dynamics (1).png</a></td></tr><tr><td><strong>Hubspot</strong></td><td>A form picker and rendering component for Hubspot forms.</td><td><a href="hubspot.md">hubspot.md</a></td><td><a href="../.gitbook/assets/hubspot.png">hubspot.png</a></td></tr><tr><td><strong>Hubspot with Umbraco Forms</strong></td><td>A custom workflow that allows form entries to be mapped to a HubSpot contact record, and then stored within the Customer Relationship Management (CRM) platform.</td><td><a href="hubspot-with-umbraco-forms.md">hubspot-with-umbraco-forms.md</a></td><td><a href="../.gitbook/assets/hubspot.png">hubspot.png</a></td></tr></tbody></table>
```

Reply:
@alina-tincas Thanks for your review, of course I can define it there as well, but I thought it was only necessary for the first time use? Please let me know 😄 

https://github.com/erikjanwestendorp/UmbracoDocs/blob/main/generic/contribute/style-guide.md#defined-acronyms

Reply:
That is correct, it is only necessary for first-time use, however, it seems that Vale still gives a warning if you check here: https://github.com/umbraco/UmbracoDocs/pull/5456/files. My thinking is that perhaps this is a table so it doesn´t catch the first-time definition 🤔

Reply:
However I can still merge it in as it is 👍

Reply:
As I can see on the documentation: https://docs.umbraco.com/umbraco-dxp/integrations/categories it seems that CRM is defined at the top of the cards 🤔 
![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/072bfab7-1388-43c1-8cb8-7f67f18436b6)

Reply:
so, in this case, it shouldn´t be necessary to make this change to add the definition for the "ActiveCampaign" card and/or "Hubspot". What do you think about this @erikjanwestendorp? 💡

Reply:
@alina-tincas I think we can close this one 😄 Add this to all cards feels a bit to much of a good thing right?

Reply:
Agreed 😊